src/platform/plugins/shared/dashboard/public/dashboard_api/types.ts

```diff
@@ -156,13 +156,14 @@ export type DashboardApi = CanExpandPanels &
     publishedChildFilters$: PublishingSubject<Filter[] | undefined>;
     unpublishedChildFilters$: PublishingSubject<Filter[] | undefined>;
     publishFilters: () => void;
+
+    layout$: BehaviorSubject<DashboardLayout>;
+    registerChildApi: (api: DefaultEmbeddableApi) => void;
```

Comment:
These needed to be moved from the internal API because `ControlsRenderer` uses both `layout$` (for its state management) and `regIsterChildApi` (so that the sticky controls count as regular children).